src/charm.py

```diff
@@ -584,6 +585,16 @@ def _update_cert(self):
         self.container.exec(["update-ca-certificates", "--fresh"]).wait()
         subprocess.run(["update-ca-certificates", "--fresh"])
 
+    def _check_disk_space(self):
+        try:
+            database_storage = self.model.storages.get('database', [])
```

Comment:
We want this runtime dep to loudly fail (we have unit tests):

```suggestion
            database_storage = self.model.storages["database"]
```